src/components/video-editor/timeline/TimelineEditor.tsx

```
@@ -1351,6 +1368,16 @@ export default function TimelineEditor({
     };
   }, [range, totalMs]);
 
+  useImperativeHandle(ref, () => ({
+    addZoom: handleAddZoom,
+    suggestZooms: handleSuggestZooms,
+    splitClip: handleSplitClip,
+    addAnnotation: handleAddAnnotation,
+    addAudio: handleAddAudio,
+    toggleCollapsed: () => {},
```

Biome (2.4.11)

[error] Unexpected empty block.

(lint/suspicious/noEmptyBlockStatements)